### PR TITLE
chore: bump yarn & docusaurus

### PR DESCRIPTION
**Pull request by layershifter**
This PR bumps `docusaurus` to latest RC. `yarn` was bumped as a consequence, it was throwing a strange error (https://github.com/yarnpkg/berry/issues/759), the latest `yarn` works as it should ¯\_(ツ)_/¯

### CODE REVIEW

File: apps/website/docusaurus.config.js [hunk 1]
```diff
@@ -44,7 +44,11 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
-      autoCollapseSidebarCategories: false,
+      docs: {
+        sidebar: {
+          autoCollapseCategories: false,
+        },
+      },
```
Comment (layershifter) [inline apps/website/docusaurus.config.js:51]: The option was renamed.

File: apps/website/src/theme/Navbar/ColorModeToggle/index.tsx [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
-import useIsBrowser from '@docusaurus/useIsBrowser';
-import { GithubIcon, MoonIcon, SunIcon } from '../../components/icons';
+import { GithubIcon, MoonIcon, SunIcon } from '../../../components/icons';
 import styles from './styles.module.css';
 
 function ThemeToggle() {
-  const disabled = useIsBrowser() === false;
-  const { isDarkTheme, setLightTheme, setDarkTheme } = useColorMode();
+  const disabled = useThemeConfig().colorMode.disableSwitch;
+  const { colorMode, setColorMode } = useColorMode();
```
Comment (layershifter) [inline apps/website/src/theme/Navbar/ColorModeToggle/index.tsx:9]: - The component was renamed
- `isDarkTheme` => `colorMode`, `setLightTheme` => `setColorMode`

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -20,7 +20,7 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: '14'
+          node-version: '16'
```
Comment (layershifter) [inline .github/workflows/ci.yml:23]: image

Docusaurus requires Node 16 now.

File: apps/website/src/theme/Navbar/ColorModeToggle/index.tsx [hunk 4]
```diff
@@ -1,24 +1,23 @@
...
 
-  const ariaLabel = isDarkTheme ? 'Switch to light mode' : 'Switch to dark mode';
-  const icon = isDarkTheme ? <SunIcon /> : <MoonIcon />;
+  const ariaLabel = colorMode === 'dark' ? 'Switch to light mode' : 'Switch to dark mode';
```
Comment (ling1726) [inline apps/website/src/theme/Navbar/ColorModeToggle/index.tsx:11]: nit: might be worth keeping `isDarkMode` as a boolean variable

**Review by ling1726 — APPROVED** [review]

**Review by bsunderhus — APPROVED** [review]

**Pull request merged**